cogs/study.py

```diff
@@ -52,6 +53,26 @@ async def ask_question(self, ctx, conversation_mode="true", *, question: str):
 
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['dict'])
+    async def urban(self, ctx, *, word):
+        """Searches urban dictionary."""
+
+        url = 'http://api.urbandictionary.com/v0/define'
+        async with ctx.session.get(url, params={'term': word}) as resp:
```

Comment:
@The-Codin-Hole/core-developers this line needs to be changed according to the new standards

Reply:
Must look like this:
```python
async with aiohttp.ClientSession() as session:
        async with session.post(API_ADDR, data=json.dumps(request)) as resp:
```